src/program.js

```diff
@@ -521,6 +521,12 @@ Example: $0 --help run.
         },
       },
     )
+    .command(
+      'dump-config',
+      'Run config discovery and dump the resulting config data',
```

Comment:
```suggestion
      'Run config discovery and dump the resulting config data as JSON',
```

(or something like that)